ci/jenkins/templates/validator-nightly-template.yaml

```
@@ -0,0 +1,27 @@
+- job-template:
+    name: 'validator-join-nightly'
+    project-type: pipeline
+    number-to-keep: 30
+    days-to-keep: 30
+    branch: master
+    wrappers:
+      - timeout:
+          timeout: 120
+          fail: true
+    triggers:
+        - timed: 'H H(3-5) * * *'
+    parameters:
+        - string:
+            name: VALIDATOR_ARGS
+            default: '-t join -n 3:2 -c'
```

Just to make it obvious, default might change
```suggestion
            default: '-p openstack -t join -n 3:2 -c'
```

---

thanks updated